### PR TITLE
fix: Gigantic feat

### LINKED ISSUE #2789: Wearable causes giant feet
### **Build version:**
Dev-latest

https://decentraland.org/marketplace/contracts/0xd2bd1cadfe12ee9d37ff95912af7f8a97ae71ba0/tokens/128

During a playtest this wearable appears to cause a problem with giant overscaled feet. We need to investigate why this is specific to this wearable and any other issues related.

### PR DESCRIPTION
**Pull request by dalkia**
## What does this PR change?

Fixes #2789 

A wearable cannot contains animators or animation because that breaks the skinning. This PR cleans up the wearable to avoid the situation

## How to test the changes?

Testing with https://decentraland.org/marketplace/contracts/0xd2bd1cadfe12ee9d37ff95912af7f8a97ae71ba0/tokens/128

1. Launch the explorer
2. Instantiate an avatar with urn `urn:decentraland:matic:collections-v2:0xd2bd1cadfe12ee9d37ff95912af7f8a97ae71ba0:0` (using the avatar creator debug)
3. Check that there are no giant feet, and the avatar has the slippers
4. Do several emotes and check that everything is okay

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md

### CODE REVIEW

**Comment by aixaCode** [comment]
How in the first place this wearable have animation or animator?

File: Explorer/Assets/DCL/AvatarRendering/Loading/Assets/AttachmentAssetUtility.cs [hunk 1]
```diff
@@ -35,8 +35,12 @@ public static CachedAttachment InstantiateWearable(this IAttachmentsAssetsCache
             {
                 Transform child = children.Value[index];
                 child.gameObject.layer = parent.gameObject.layer;
-            }
 
+                //Wearables shouldnt have animators or animations since it will break the skinning
+                Object.Destroy(child.GetComponent<Animator>());
```
Comment (mikhail-dcl) [inline Explorer/Assets/DCL/AvatarRendering/Loading/Assets/AttachmentAssetUtility.cs:40]: Is it safe to delete a component if it is not attached?

Comment (dalkia) [reply Explorer/Assets/DCL/AvatarRendering/Loading/Assets/AttachmentAssetUtility.cs:40]: I had the same doubt. I did a very simple test script, and destroying a `null` reference does not throw an exception. Looks like we are safe

Comment (dalkia) [reply Explorer/Assets/DCL/AvatarRendering/Loading/Assets/AttachmentAssetUtility.cs:40]: The incredible test:

```
        try
        {
            Object.Destroy(GetComponent<Animator>());
        }
        catch (Exception e)
        {
            Debug.Log($"Exception {e}");
        }
```

The GameObject didn't have an animator, and an exception was not thrown

**Comment by dalkia** [comment]
> How in the first place this wearable have animation or animator?

The creator made it that way. I guess I could try and remove it at AB converter level, but I don't know how many of them may have this issue still

**Review by m3taphysics — APPROVED** [review]

**Pull request merged**